### PR TITLE
add fix and updated changes.md with details

### PR DESCRIPTION
**Pull request by setswei**
Added fix to allow the LM Sensors to work again

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 flask==2.0.3
 requests
 logicmonitor_sdk==2.0.163
+werkzeug==2.2.2
```
Comment (arm4b) [inline requirements.txt]: Yeah, you can add Python 3.8 in the workflow like this:
https://github.com/StackStorm-Exchange/stackstorm-openai/blob/6e492e6b945fe8b77c8a3516d965a2904383f90e/.github/workflows/build_and_test.yaml#L12-L17
There is also a release workflow in the same dir that needs to be updated too.

Though with dropping python 3.6, this pack probably needs a major release bump like `2.0.0`.

But before that, from the https://werkzeug.palletsprojects.com/en/2.2.x/changes/#version-2-0-3, `2.0.3` is the latest version that supported python 3.6.
Can we try that first?

Comment (arm4b) [reply requirements.txt]: cc @lm-ydubler

Comment (setswei) [reply requirements.txt]: werkzeug v2.0.3 does work. I have also tested this on my local docker env and production environment and the pack is working as expected. I am happy for the 2.0.3 to be merged in for now and we can look at a major release update later on. That way LogicMonitor Developers can have their say as well

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v1.2.0
+
+* Due to upstream issues with werkzeug url dependency (a recent release has caused issues). This issue resulted in the
+  LogicMonitor Sensor to consistently crash. werkzeug==2.0.3 has been added to the requirements.txt file. This is the
+  last known working version of the dependency which has fixed the issue with the LogicMonitor Sensor
```
Comment (arm4b) [inline CHANGES.md]: Let's rephrase it to something like this:

```suggestion
* Fix LogicMonitor Sensor crash caused by updated upstream dependencies by pining werkzeug==2.0.3
```

^^ or similar for a more clear and less lengthy description

File: pack.yaml [hunk 3]
```diff
@@ -6,7 +6,7 @@ keywords:
   - LogicMonitor
   - monitoring
   - alerting
-version: 1.1.0
+version: 1.2.0
```
Comment (arm4b) [inline pack.yaml]: Since this PR is not changing anything major and it's just a bugfix, bumping a patch version would be more appropriate

```suggestion
version: 1.1.1
```

See https://semver.org/#spec-item-6

**Review by arm4b — CHANGES_REQUESTED** [review]
All good!

Let's just bump the patch version instead then, considering the fix now is not a breaking change anymore.

Should be easily mergeable after that.

File: requirements.txt (hunk 1 above)
Comment (arm4b) [reply requirements.txt]: Nice! Left a few final minor comments

**Comment by setswei** [comment]
Sorry @armab, I have been traveling for work. I have made the updates you requested.

**Comment by setswei** [comment]
Just checking to see if this can still be merged in. i did update the version number as requested

**Review by nzlosh — APPROVED** [review]
